### PR TITLE
Demos for PayID / Xpring in Java

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

Demos for PayID / Xpring components in Java.

### Context of Change

Adding demos to show off our new components.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

More demos!

## Test Plan

CI updated to prove everything still builds.

### CODE REVIEW

File: java/src/main/java/io/xpring/demo/PayIdDemo.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    // The BTC network to resolve on.
+    String btcNetwork = "btc-testnet";
+
+    // A client to resolve PayIDs on the XRP Ledger.
+    XRPPayIDClient xrpPayIdClient = new XRPPayIDClient(xrpNetwork);
+
+    // A client to resolve PayIDs on the Bitcoin network.
+    PayIDClient btcPayIdClient = new PayIDClient(btcNetwork);
+
+    System.out.println("Resolving Pay ID: " + payId);
```
Comment (amiecorso) [inline java/src/main/java/io/xpring/demo/PayIdDemo.java:26]:
```suggestion
    System.out.println("Resolving PayID: " + payId);
```

File: java/src/main/java/io/xpring/demo/PayIdDemo.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    System.out.println("On network: " + xrpNetwork);
+
+    String xrpAddress = xrpPayIdClient.xrpAddressForPayID(payId);
+    System.out.println("Resolved to " + xrpAddress);
+    System.out.println("");
+
+    System.out.println("Resolving Pay ID: " + payId);
```
Comment (amiecorso) [inline java/src/main/java/io/xpring/demo/PayIdDemo.java:33]:
```suggestion
    System.out.println("Resolving PayID: " + payId);
```

File: java/src/main/java/io/xpring/demo/XpringDemo.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    // The expected address of the gRPC server.
+    String grpcURL = "test.xrp.xpring.io:50051";
+
+    // A wallet with funds on testnet.
+    Wallet wallet = new Wallet("snYP7oArxKepd3GPDcrjMsJYiJeJB", true);
+
+    // The number of drops to send.
+    BigInteger dropsToSend = BigInteger.valueOf(10);
+
+    // The Pay ID to resolve.
```
Comment (amiecorso) [inline java/src/main/java/io/xpring/demo/XpringDemo.java:25]:
```suggestion
    // The PayID to resolve.
```

**Review by amiecorso — APPROVED** [review]
Again just a note on brand consistency.  I ran these demos locally and it worked for me (no warnings!)

**Review by LoisRP — APPROVED** [review]
Agree with Amie's suggestions.

**Review by amiecorso — APPROVED** [review]

**Pull request merged**